settings.sample.ini

```diff
@@ -0,0 +1,49 @@
+; This is a sample settings file. Copy it to "settings.ini" and configure it
+; before running make.
+
+; To load variables in bash use:
+; > source $current_dir/common.sh
+; > declare -A LIST_OF_APPS
+; > get_settings_section settings.ini fdroid_apps LIST_OF_APPS
+
+
+; General settings.
+[general]
+system_app_destination=/system/priv-app ; no trailing slash.
+user_app_destination=/data/app ; no trailing slash.
+;password_file=/path/to/secret/file
+
+; Preferred snapshot version
+snapshot_version=M12
+
+
+; Installed before those downloaded from the f-droid.org repository.
+[apps]
+FDroid=https://f-droid.org/FDroid.apk
+OrfoxFennec-debug=https://guardianproject.info/builds/OrfoxFennec/latest/OrfoxFennec-debug.apk
+
+
+; Contains a list of applications to be installed.
+; If you feel adventurous use "latest" to retrieve the latest released version.
+; NOTE: Versions are locked because we want to be sure that the apps are known
+;       to be secure.
+[fdroid_apps]
```

Comment:
We lost the official mozilla firefox pkg (we just have the custom, bleeding-edge fdroid build). It just so happens that the one we lost is the only working browser atm (fdroid fennec and orfox are crashing)